test/e2e/util_gatewayapi_test.go

```diff
@@ -598,15 +601,15 @@ func assertHttpRouteSuccessful(t *testing.T, namespace, name string, gateway *ga
 	// Wait 1 minute for parent/s to update
 	err := wait.PollUntilContextTimeout(context.Background(), 1*time.Second, 1*time.Minute, false, func(context context.Context) (bool, error) {
 		if err := kclient.Get(context, nsName, httproute); err != nil {
-			t.Logf("failed to get httproute %s/%s, retrying...", namespace, name)
+			t.Logf("Failed to get httproute %v: %v", nsName, err)
```

Comment:
A couple more cases here that had the "retrying" before.  Also, I expect that the error message is not going to change much on each iteration.

Reply:
Added back "retrying" in https://github.com/openshift/cluster-ingress-operator/compare/77beec1a8b6af8c8c5ce00f29f7a60683a84cd75..92e89a2a02fccb3d149e3e0491a30f19fe160774.

I don't think it's worth the effort to save and compare the last `err.String()` to make logging slightly more compact.  If it is worth that effort, then we should probably write a `getWithRetry` helper and update all tests to use it.